content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md

```diff
@@ -269,7 +281,99 @@ This returns the same output as checking the flow status on Swagger UI. You see:
 
 If you have any questions about the Corda Node CLI commands, run the `--help` command for more information.
 
+## Set up and test the UI
+
+The Solar System CorDapp comes with a built-in UI. See the <a href="https://github.com/corda/samples-kotlin-corda5/tree/main/Tutorial/solarsystem/Web-UI">`Web-UI`</a> folder of the sample folder to study the code.
+
+### Before you start
+
+Before you can build the Solar System CorDapp UI, you must:
+
+* [Deploy your CorDapp to a local Corda 5 network](*deploy-the-cordapp-using-corda-cli)
+* Download the [Node.js](https://nodejs.org/en/download/) asynchronous event-driven JavaScript runtime for your platform.
```

Comment:
We should probably just give them _a_ version to use

Reply:
Good idea. Added this in. Thanks!